impala/_thrift_api.py

```diff
@@ -346,23 +402,37 @@ def get_http_transport(host, port, http_path, timeout=None, use_ssl=False,
         except AttributeError:
             auth = base64.encodestring(user_password).decode().strip('\n')
 
-        transport.setCustomHeaders({'Authorization': 'Basic %s' % auth})
+        def get_custom_headers(cookie_headers, has_auth_cookie):
```

Comment:
nit: 'cookie_header' would be a better name here too.

Reply:
fixed